Error: Something happened
	at try-catch-finally.b:3:3 -> @.script()
Cleaning up